torch/ao/quantization/qconfig_mapping.py

```diff
@@ -106,15 +106,12 @@ def _get_default_qconfig_mapping(is_qat: bool, backend: str, version: int) -> QC
             fixed_qparams_observer_to_qconfig[observer] = fixed_qparams_qconfig
         qconfig_mapping.set_object_type(fixed_qparams_op, fixed_qparams_qconfig)
 
-    # QConfig for fused ops for onednn backend
+    # Use global qconfig for tanh to support linear - tanh fusion
     # Separate ops are required to have the same qconfig as fused ops
     # TODO: we should be able to configure qconfig for patterns
     if backend == 'onednn':
-        qconfig_mapping.set_object_type(torch.nn.Linear, qconfig) \
-                       .set_object_type(torch.nn.LeakyReLU, qconfig) \
-                       .set_object_type(torch.nn.functional.leaky_relu, qconfig) \
-                       .set_object_type(torch.nn.Tanh, qconfig) \
-                       .set_object_type(torch.nn.functional.tanh, qconfig)
+        qconfig_mapping.set_object_type(torch.nn.Tanh, None) \
```

Comment:
Does `set qconfig to None` mean we use `FP32` for `Tanh`? I thought we should use same qconfig for `linear` and `Tanh` to enable the fusion, doesn't it?

Reply:
right, None means don't quantize

Reply:
Thanks. Good point. I thought `None` meant global and I meant to use global `qconfig` for `tanh` so that `tanh` and `qlinear` use the same `qconfig`. Now that it was wrong and did not work, I have to find a new way to fix it.
It is required that `QConfig` must specify a `FixedQParamsObserver` or a `FixedQParamsFakeQuantize` for fixed qparams ops (e.g. tanh). https://github.com/pytorch/pytorch/blob/ad782ff7df950dfda64e271dd4ee3c6128971103/torch/ao/quantization/fx/utils.py#L811
@jerryzh168 Do you know any solution or workaround? Thanks!

Reply:
Looks like there is no easy way to set correct `qconfig_mappings` for both `tanh` and `linear - tanh` fusion. I removed these lines. Users will need to set `qconfig` manually if they want to fuse `linear - tanh` with onednn backend.